### PR TITLE
Feature New File

### PR DESCRIPTION
**Pull request by dixonyant**

### CODE REVIEW

**Review by shantahuja — COMMENTED** [review]
why are u so ceool?

File: stc_game/Assets/new file!!.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+jfasdkasdjf
+
+
+
+
+weioraweiojai
```
Comment (dixonyant) [inline stc_game/Assets/new file!!.txt:6]: Why did you add this?

File: stc_game/Assets/new file!!.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+jfasdkasdjf
```
Comment (dixonyant) [inline stc_game/Assets/new file!!.txt:1]: this has reference

**Review by tcape — COMMENTED** [review]
this is weird

**Review by shantahuja — CHANGES_REQUESTED** [review]
dsfajll;jfdka;dka

**Review by tcape — CHANGES_REQUESTED** [review]
request change
